ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,50 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
```

Comment:
if `fPass` is a formula for doing nothing, I suggest `fIdentity` or `fBypass`.

Reply:
What about calling it `fEmptyResult` since the `result` dictionary will be empty?

Reply:
Oh I thought it just bypasses the input data since the original name was pass but like you said, it receives anything and return nothing. Maybe fEmptyResult would be more proper then.

Reply:
This is now in #376.